android/beagle/src/main/java/br/com/zup/beagle/android/cache/imagecomponent/LruImageCache.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package br.com.zup.beagle.android.cache.imagecomponent
+
+import android.graphics.Bitmap
+import android.util.LruCache
+
+class LruImageCache(private val memoryCache: LruCache<String, Bitmap> =
+                            LruCache<String, Bitmap>(anEighthOfMemory())) {
+
+    fun put(key: String, bitmap: Bitmap) {
+        memoryCache.put(key, bitmap)
+    }
+
+    fun get(key: String?): Bitmap? = memoryCache.get(key)
+
+    private object HOLDER {
```

Comment:
Follow the guideline name provided by kotlin to created objects and classes:
https://kotlinlang.org/docs/reference/coding-conventions.html

```suggestion
    private object Holder {
```

Reply:
Done!